server/eclipse-project/src/main/java/au/org/democracydevelopers/corla/model/vote/IRVBallotInterpretation.java

```diff
@@ -1,25 +1,146 @@
+ * auditing logic. Every time an invalid IRV vote is encountered, whether in the initial upload or
+ * during the auditing steps, a record is made of the CVR ID, the county, contest name, raw vote,
+ * and our interpretation of that vote as an ordered list of candidate names.
+ * This is output as a report called IRVBallotInterpretationReport.
+ */
+@Entity
+@Table(name = "irv_ballot_interpretation")
+public class IRVBallotInterpretation implements PersistentEntity, Serializable {
 
-The colorado-rla IRV extensions are distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
-without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
-See the GNU Affero General Public License for more details.
+  /**
+   * ID for interpretation record (for persistence).
+   */
+  @Id
+  @Column(updatable = false, nullable = false)
+  @GeneratedValue(strategy = GenerationType.SEQUENCE)
+  private Long id;
 
-You should have received a copy of the GNU Affero General Public License along with
-raire-service. If not, see <https://www.gnu.org/licenses/>.
-*/
+  /**
+   * The version (for optimistic locking).
+   */
+  @Version
+  private Long version;
 
-package au.org.democracydevelopers.corla.model.vote;
 
-public class IRVBallotInterpretation {
-}
+  /**
+   * The contest to which the interpreted vote belongs.
+   */
+  @ManyToOne(optional = false, fetch = FetchType.LAZY)
+  private Contest contest;
+
+  /**
+   * The record type, either UPLOADED or AUDITOR_ENTERED.
+   */
+  @Column(name = "record_type", nullable = false)
+  @Enumerated(EnumType.STRING)
+  private CastVoteRecord.RecordType recordType;
+
+  /**
+   * ID of the CVR being audited, and whose paper ballot is being interpreted.
+   * There may be multiple records with the same CVR ID, if one ballot contains multiple IRV contests.
+   */
+  @Column(name = "cvr_number", nullable = false)
+  private int cvrNumber;
+
+  /**
+   * The imprinted ID, generally tabulator_id-batch_id-record_id.
+   */
+  @Column(name = "imprinted_id", nullable = false)
+  private String imprintedID;
+
+  /**
+   * List of candidate names with ranks in parentheses representing a raw vote in an IRV contest.
+   * Order is not important.
+   */
+  @Column(name = "raw_choices", columnDefinition = "character varying (1024)", nullable = false)
+  @Convert(converter = StringListConverter.class)
+  private List<String> rawChoices = new ArrayList<>();
+
+  /**
+   * List of candidates names, in order of rank, representing a valid vote in an IRV contest.
+   */
+  @Column(name = "interpretation", columnDefinition = "character varying (1024)", nullable = false)
+  @Convert(converter = StringListConverter.class)
+  private List<String> interpretation = new ArrayList<>();
+
+  /**
+   * Construct an empty IRVBallotInterpretation (for persistence).
+   */
+  public IRVBallotInterpretation() {
+  }
+
```

Comment:
Worth adding in the comments that we are creating a record of an interpretation of an invalid IRV vote.